rhizome/common/bin/daemonizer

```diff
@@ -65,7 +65,7 @@ else
       command = "#{command} < #{dir}/var/proc/#{name}.stdin"
     end
 
-    command = "/bin/bash -c '#{command} > #{dir}/var/log/#{name}.stdout 2> #{dir}/var/log/#{name}.stderr'"
+    command = "/bin/bash -c '#{command}'"
```

Comment:
On balance, I find the journal more useful than the /var/log files that get truncated and overwritten with the latest output all the time. I was debugging something at the time. Though, I could see a combined system that retained the _first_ invocation's stdout/stderr this way, perhaps the last one, and spooled the rest to journal. But journalctl is likely enough.